src/components/ConfirmTermination/ConfirmTermination.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { ConfirmDialog } from '@amsterdam/bmi-component-library';
+import { BackDropStyle } from './ConfirmTerminationStyles';
+import { createPortal } from 'react-dom';
+
+export interface IState {
+	message: string;
+	onConfirm: () => void;
+	title?: string;
+	textCancelButton?: string;
+	textConfirmButton?: string;
+	onCancel?: () => void;
+	size?: string;
+	onClose?: () => void;
+}
+
+export type Props = {
+	size?: 'xs' | 'sm' | 'md' | 'lg' | 'xl';
```

Comment:
Not required but you could have consider to make this an enum to make it type safe across components

Reply:
Alternatively:
```
type ModalSizes = 'xs' | 'sm' | ...
```

Reply:
Converted to enum and extracted this to a seperate file